benchmark/opperf/nd_operations/gemm_operators.py

```diff
@@ -55,33 +55,62 @@ def run_gemm_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler='nativ
 
```

Comment:
Please add the large_tensor parameter in description

Reply:
Good catch, will do.